dev/create-release/releaseutils.py

```diff
@@ -49,6 +49,11 @@
     print("Install using 'sudo pip install unidecode'")
     sys.exit(-1)
 
+try:
+    raw_input          # Python 2
```

Comment:
and shall we do this with if-else of Python version to be consistent with other places?

Reply:
“Where practical, apply the Python porting best practice: [Use feature detection instead of version detection](https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection).”

Reply:
Shall we keep this consistent for now? There are a hell of a lot things to fix in PySpark in that way.